src/pfe/portal/modules/User.js

```diff
@@ -257,7 +247,7 @@ module.exports = class User {
           let settingsFilePath = path.join(projFile.workspace, projName, '.cw-settings');
           const settFileExists = await fs.pathExists(settingsFilePath);
           const settFile = settFileExists ? await fs.readJson(settingsFilePath) : {};
-          const project = new Project({ ...projFile, ...settFile });
+          const project = new Project({ ...projFile, ...settFile }, this);
```

Comment:
Is passing `this` to a new Project object out of data here and on line 363?
It looks like you changed to pass the user object to `loadrunner.runLoad` on line 166.